script_01.py

```diff
@@ -0,0 +1,71 @@
+import math
+import cmath
+
+def task1(text="Count stuff in this random sentence."):
+    print(text)
+    vowels=['a','e','i','o','u','y']
```

Comment:
please consider this document while reviewing your code with regard to whistespaces:
https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Reply:
considered